File: sparql/src/main/java/org/aksw/commons/jena/sparql/SampleSparqlEndpoint.java
package org.aksw.commons.jena.sparql;

import java.util.HashSet;
import java.util.Set;

/**
 * Created by Sebastian Hellmann - http://bis.informatik.uni-leipzig.de/SebastianHellmann
 * Date: 13.03.11
 */
public class SampleSparqlEndpoint {

    public static void main(String[] args) {

        //Set("http://dbpedia.org")
        Set<String> s = new HashSet<String>();
       // HttpSparqlEndpoint coreEndpoint = new HttpSparqlEndpoint("http://dbpedia.org/sparql", s);

        //val r = coreEndpoint.executeSelect( "Select ?s ?p ?o From <http://dbpedia.org> { ?s ?p ?p . } Limit 10")

        //print(r)

    }
}

File: sparql-scala/src/main/scala/org/aksw/commons/sparql/sample/SampleSparqlEndpoint.scala
package org.aksw.commons.sparql.core.sample

import org.aksw.commons.sparql.core.impl.HttpSparqlEndpoint
import org.aksw.commons.sparql.core.WithoutLikeSupport

/**
 * @author Sebastian Hellmann <hellmann@informatik.uni-leipzig.de>
 * just a sample implementation
 * for your convenience ... 
 */

object SampleSparqlEndpoint {

  def main(args: Array[String]): Unit = {
    val coreEndpoint = new HttpSparqlEndpoint("http://dbpedia.org/sparql", Set("http://dbpedia.org"))  with WithoutLikeSupport


    //val r = coreEndpoint.executeSelect( "Select ?s ?p ?o From <http://dbpedia.org> { ?s ?p ?p . } Limit 10")

    println(coreEndpoint.like("test", Set("aa", "bb")))


  }
}